code/__DEFINES/mob.dm

```diff
@@ -11,6 +11,11 @@
 #define MOB_SUPERNATURAL 32
 #define MOB_NO_PETRIFY 64 //can't get petrified
 
+// For blob_act
+
+#define BLOB_ACT_WEAK 3
```

Comment:
blob_act is in no way a mob proc
or is it here because the blob is a mob? this seems out of place

Reply:
Will move it to someplace better then